code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -145,7 +145,11 @@ var/list/ai_list = list()
 			job = "AI"
 	ai_list += src
 	..()
-	playsound(src, 'sound/machines/WXP_startup.ogg', 75, FALSE)
+	var/windowserror = pick('sound/machines/WXP_error.ogg', 'sound/machines/W95_error.ogg', 'sound/machines/W98_error.ogg', 'sound/machines/W7_error.ogg')
```

Comment:
Put it in the lists of sound in sound.dm

Reply:
the what now

Reply:
https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/game/sound.dm
make it a list like these